waku/v0/peer.go

```diff
@@ -173,6 +173,10 @@ func (p *Peer) EnodeID() enode.ID {
 	return p.p2pPeer.ID()
 }
 
+func (p *Peer) Name() string {
+	return p.p2pPeer.Name()
+}
+
```

Comment:
🙂 Updating a deprecated API to ensure compatibility with the Peer interface.

I don't have a better solution.